content/en/docs/concepts/policy/node-resource-managers.md

```diff
@@ -189,11 +189,15 @@ The following policy options exist for the static CPU management policy:
 `align-by-socket` (alpha, hidden by default)
 : Align CPUs by physical package / socket boundary, rather than logical NUMA boundaries (available since Kubernetes v1.25)
 `distribute-cpus-across-cores` (alpha, hidden by default)
-: allocate virtual cores, sometimes called hardware threads, across different physical cores  (available since Kubernetes v1.31)
+: Allocate virtual cores, sometimes called hardware threads, across different physical cores  (available since Kubernetes v1.31)
 `distribute-cpus-across-numa` (alpha, hidden by default)
-: spread CPUs across different NUMA domains, aiming for an even balance between the selected domains (available since Kubernetes v1.23)
+: Spread CPUs across different NUMA domains, aiming for an even balance between the selected domains (available since Kubernetes v1.23)
 `full-pcpus-only` (beta, visible by default)
 : Always allocate full physical cores (available since Kubernetes v1.22)
+`strict-cpu-reservation` (alpha, hidden by default)
+: Prevent all the pods regardless of their Quality of Service class to run on reserved CPUs (available since Kubernetes v1.32)
```

Comment:
Same, fixing a rebase artifact of my previous https://github.com/kubernetes/website/pull/48797